cmd/sidecars/network-passt-binding/server/server.go

```diff
@@ -91,7 +92,17 @@ func (s V1alpha3Server) OnDefineDomain(
 		IstioProxyInjectionEnabled: istioProxyInjectionEnabled,
 	}
 
-	passtConfigurator, err := domain.NewPasstNetworkConfigurator(vmi.Spec.Domain.Devices.Interfaces, vmi.Spec.Networks, opts, nil)
+	primaryPodIfaceName, err := primaryPodIfaceNameFromVMI(vmi)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
This is just continuing the old comment thread [here](https://github.com/kubevirt/kubevirt/pull/15311/commits/26800e74c9f0a423f0db7a90837b7c71a5aceaab#r2242488708).

Can you explain why you have not added the needed to `NewPasstNetworkConfigurator`? It has many of the steps you do in the new function helper.